prep-javascript-variables/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName = 'James Aguirre';
+    console.log(fullName);
+    console.log(typeof fullName);
+    numberOfPets = 1;
```

Comment:
don't forget to declare this variable, you don't want JavaScript to declare it for you.